bbp/default.nix

```diff
@@ -221,6 +221,12 @@ let
         bluepy = callPackage ./nse/bluepy {
         };
 
+        bluepy_0_11_2 = callPackage ./nse/bluepy {
```

Comment:
Ok, Can you make the new one the default and the old one the versionned one ( bluepy_0_X )

Or I already fear that the scientist will only load the old one without wanting it :)